atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1642,9 +1642,18 @@ private void putMetadataForTablesInternal(
     }
 
     private void putMetadataWithoutChangingSettings(final TableReference tableRef, final byte[] meta) {
+        long ts = System.currentTimeMillis();
```

Comment:
Out of interest what is the value of using system time, I guess it would only matter if  a schema change coincided with a big clock shift.